explorer/lib/explorer/models/restakings.ex

```diff
@@ -29,6 +29,7 @@ defmodule Restakings do
     Operators.get_operators()
       |> Enum.map(fn operator -> StakeRegistryManager.has_operator_changed_staking(%{fromBlock: from_block, operator_id: operator.id, operator_address: operator.address}) end)
       |> Enum.reject(fn {_operator_id, _operator_address, has_changed_stake} -> not has_changed_stake end)
+      |> Enum.reject(fn {operator_id, _operator_address, _has_changed_stake} -> not Operators.get_operator_by_id(operator_id).is_active end)
```

Comment:
Could this be done in the same function call? Like:
```elixir
|> Enum.reject(fn {operator_id, _operator_address, has_changed_stake} -> not has_changed_stake or not Operators.get_operator_by_id(operator_id).is_active end)
```

Reply:
It would work but it would be less efficient because `Operators.get_operator_by_id()` would be called for every operator